crates/lgn-graphics-renderer/src/core/render_object.rs

```diff
@@ -315,7 +325,19 @@ struct FatPtr {
 #[allow(unsafe_code)]
 fn get_fat_ptr<P, S>(data: Box<dyn SecondaryTableHandler<P, S>>) -> FatPtr {
     // Make a FatPtr from the Box. Implicitly forgotten, must be properly dropped in the SecondaryTable drop impl.
-    unsafe { transmute::<_, FatPtr>(Box::into_raw(data)) }
+    unsafe {
+        let p = Box::into_raw(data);
```

Comment:
This is just to let the type be apparent, so we know what to transmute into in drop_fat_ptr below.